src/main/java/edu/hm/hafner/analysis/Report.java

```diff
@@ -783,6 +788,88 @@ public int hashCode() {
         result = 31 * result + duplicatesSize;
         return result;
     }
+    
+    private void writeObject(ObjectOutputStream oos) throws IOException {
+        oos.defaultWriteObject();
```

Comment:
It would make sense to use a simple custom format for the remaining lists as well. Then you do not need the trick with the `transient` property.